packages/eslint-config-udemy-basics/README.md

```diff
@@ -33,4 +33,4 @@ module.exports = {
 
 # Contributing
 
-See [Contributing](/#contributing).
\ No newline at end of file
+See [Contributing](/README.md#contributing).
```

Comment:
Weirdly enough not explicitly stating README.md did not work for this absolute-like relative path syntax.